lua/entities/gmod_wire_expression2/core/custom/cl_clipboard.lua

```diff
@@ -0,0 +1,5 @@
+E2Helper.Descriptions["setClipboardText(s)"] = "Adds the given string to the computers clipboard. Can not exceed 65532 characters"
```

Comment:
Should be a part of the wider e2helper file, and as denneisk said as a part of an existing extension rather than a new file. I'd go for console, debug or core.